Anchorage.swift

```diff
@@ -33,139 +33,141 @@ extension NSLayoutDimension : LayoutAnchorType {}
 }
 
 @discardableResult public func >=<T: NSLayoutDimension>(lhs: T, rhs: T) -> NSLayoutConstraint {
-    return activateConstraint(lhs.constraint(greaterThanOrEqualTo: rhs))
+    return activate(constraint: lhs.constraint(greaterThanOrEqualTo: rhs))
 }
 
 @discardableResult public func >=<T: NSLayoutXAxisAnchor>(lhs: T, rhs: T) -> NSLayoutConstraint {
-    return activateConstraint(lhs.constraint(greaterThanOrEqualTo: rhs))
+    return activate(constraint: lhs.constraint(greaterThanOrEqualTo: rhs))
 }
 
 @discardableResult public func >=<T: NSLayoutYAxisAnchor>(lhs: T, rhs: T) -> NSLayoutConstraint {
-    return activateConstraint(lhs.constraint(greaterThanOrEqualTo: rhs))
+    return activate(constraint: lhs.constraint(greaterThanOrEqualTo: rhs))
 }
 
 @discardableResult public func >= <T: NSLayoutDimension>(lhs: T, rhs: LayoutExpression<T>) -> NSLayoutConstraint {
-    return activateConstraint(lhs.constraint(greaterThanOrEqualTo: rhs.anchor!, constant: rhs.constant), withPriority: rhs.priority)
+    return activate(constraint: lhs.constraint(greaterThanOrEqualTo: rhs.anchor!, constant: rhs.constant), withPriority: rhs.priority)
 }
 
 @discardableResult public func >= <T: NSLayoutXAxisAnchor>(lhs: T, rhs: LayoutExpression<T>) -> NSLayoutConstraint {
-    return activateConstraint(lhs.constraint(greaterThanOrEqualTo: rhs.anchor!, constant: rhs.constant), withPriority: rhs.priority)
+    return activate(constraint: lhs.constraint(greaterThanOrEqualTo: rhs.anchor!, constant: rhs.constant), withPriority: rhs.priority)
 }
 
 @discardableResult public func >= <T: NSLayoutYAxisAnchor>(lhs: T, rhs: LayoutExpression<T>) -> NSLayoutConstraint {
-    return activateConstraint(lhs.constraint(greaterThanOrEqualTo: rhs.anchor!, constant: rhs.constant), withPriority: rhs.priority)
+    return activate(constraint: lhs.constraint(greaterThanOrEqualTo: rhs.anchor!, constant: rhs.constant), withPriority: rhs.priority)
 }
 
 @discardableResult public func >= (lhs: NSLayoutDimension, rhs: LayoutExpression<NSLayoutDimension>) -> NSLayoutConstraint {
     if let anchor = rhs.anchor {
-        return activateConstraint(lhs.constraint(greaterThanOrEqualTo: anchor, multiplier: rhs.multiplier, constant: rhs.constant), withPriority: rhs.priority)
+        return activate(constraint: lhs.constraint(greaterThanOrEqualTo: anchor, multiplier: rhs.multiplier, constant: rhs.constant), withPriority: rhs.priority)
     }
     else {
-        return activateConstraint(lhs.constraint(greaterThanOrEqualToConstant: rhs.constant), withPriority: rhs.priority)
+        return activate(constraint: lhs.constraint(greaterThanOrEqualToConstant: rhs.constant), withPriority: rhs.priority)
     }
 }
 
 @discardableResult public func >= (lhs: EdgeAnchors, rhs: EdgeAnchors) -> EdgeConstraints {
-    return lhs.activeConstraintsGreaterThanOrEqualToEdges(rhs)
+    return lhs.activeConstraints(greaterThanOrEqualToEdges: rhs)
```

Comment:
There's a slight naming inconsistency between `activeConstraints` and `activate(constraint` -- both activate vs active and the start of the tag. I think it should be `activate(constraintsGreaterThanOrEqualTo:`